project/Version.scala

```diff
@@ -4,8 +4,8 @@ import sbt._
 object Version {
   val baseModelVersion = "0.1"
 
-  def getVersionString(baseDir: File) = {
-    def getLastModelCommitFromGit = { s"""git log -n 1 --pretty=format:%h ${baseDir.getAbsolutePath}""" !! }
```

Comment:
With the old code, each of rawls-utils, workbench-google, workbench-util would get published with different git hashes (the hash of the last change in the subdir). I found it too confusing. Now they all get pushed with the same hash.